module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.hpp

```diff
@@ -122,6 +124,39 @@ namespace module::behaviour::strategy {
                   const message::support::FieldDescription& fieldDescription,
                   const message::input::GameState::Data::Mode& mode);
 
+        void penaltyShootout(const message::input::GameState& gameState,
+                             message::input::GameState::Data::Phase phase,
```

Comment:
Does it work? (same as Ysobel's suggestion)
```suggestion
                             const message::input::GameState::Data::Phase& phase,
```

Reply:
Yes.